src/thunderbird_accounts/authentication/permissions.py

```diff
@@ -122,3 +128,16 @@ def authenticate(self, request):
 
         # Return user and decoded_jwt for request.user and request.auth
         return user, decoded_jwt
+
+
+class IsValidPaddleWebhook(BaseAuthentication):
+    def authenticate(self, request):
+        if not Verifier or not Secret:
+            logging.error('Paddle package is not installed. This webhook has been rejected.')
+            return None
+
+        integrity_check = Verifier().verify(request, Secret(settings.PADDLE_WEBHOOK_KEY))
+
+        # We need to return a user, but we don't need the user for these requests
+        # So return an empty user object
+        return User(), None if integrity_check else None
```

Comment:
`None if integrity_check else None`

Is the first `None` a typo?

Reply:
I take this as it will always return `User()` but only return a second argument on condition (but always returns `None`)

Reply:
Good catch, that should be the first argument!